src/Components/MainTableView/DataStorageClasses/TableAttribute.tsx

```diff
@@ -68,218 +70,246 @@ class TableAttribute {
+
       }
       else if (tableAttribute.attributeType === TableAttributeType.VAR_CHAR) {
-        return (
-          <div className="fieldUnit" key={JSON.stringify(tableAttribute)}>
-            <input type='text' value={currentValue} defaultValue={defaultValue} id={tableAttribute.attributeName} onChange={handleChange.bind(this, tableAttribute.attributeName)}></input>
-          </div>
-        );
+        return <input type="text" value={currentValue} defaultValue={defaultValue} id={tableAttribute.attributeName} onChange={(e) => handleChange(e, tableAttribute.attributeName)}></input>
       }
       else if (tableAttribute.attributeType === TableAttributeType.UUID) {
-        return (
-          <div className="fieldUnit" key={JSON.stringify(tableAttribute)}>
-            <input type='text' value={currentValue} defaultValue={defaultValue} id={tableAttribute.attributeName} onChange={handleChange.bind(this, tableAttribute.attributeName)}></input>
-          </div>
-        );
+        return <input type="text" value={currentValue} defaultValue={defaultValue} id={tableAttribute.attributeName} onChange={(e) => handleChange(e, tableAttribute.attributeName)}></input>
+
       }
       else if (tableAttribute.attributeType === TableAttributeType.DATE) {
-        return (
-          <div className="fieldUnit" key={JSON.stringify(tableAttribute)}>
-            <input type='date' defaultValue={defaultValue} id={tableAttribute.attributeName} onChange={handleChange.bind(this, tableAttribute.attributeName)}></input>
-          </div>
-        )
+        return <input type="date" defaultValue={defaultValue} id={tableAttribute.attributeName} onChange={(e) => handleChange(e, tableAttribute.attributeName)}></input>
       }
-      else if (tableAttribute.attributeType === TableAttributeType.DATETIME) {
-        return (
-          <div className="fieldUnit" key={JSON.stringify(tableAttribute)}>
-            <div className="dateTimeFields">
-              <input type='date' defaultValue={defaultValue} id={tableAttribute.attributeName + '__date'} onChange={handleChange.bind(this, tableAttribute.attributeName + '__date')}></input>
-              <input type='time' step="1" defaultValue={defaultValue} id={tableAttribute.attributeName + '__time'} onChange={handleChange.bind(this, tableAttribute.attributeName + "__time")}></input>
-            </div>
+      else if (tableAttribute.attributeType === TableAttributeType.DATETIME || tableAttribute.attributeType === TableAttributeType.TIMESTAMP) {
+        return(
+          <div className="dateTimeFields">
+            <input type="date" defaultValue={defaultValue} id={tableAttribute.attributeName + "__date"} onChange={(e) => handleChange(e, tableAttribute.attributeName + "__date")}></input>
+            <input type="time" step="1" defaultValue={defaultValue} id={tableAttribute.attributeName + "__time"} onChange={(e) => handleChange(e, tableAttribute.attributeName + "time")}></input>
```

Comment:
onChange for the time input field probably should be
```suggestion
            <input type="time" step="1" defaultValue={defaultValue} id={tableAttribute.attributeName + "__time"} onChange={(e) => handleChange(e, tableAttribute.attributeName + "__time")}></input>
```

Reply:
opps screwed on the renaming. thanks nice catch